docs/handbook/getting-started.md

```diff
@@ -580,6 +575,15 @@ Replace "X" with the appropriate letter for your USB device
 
 ### illumos/Solaris
 
+
+#### Method 1 (New releases)
+
+```bash
+sudo dd bs=4M if=./OI_MATE_experimental.usb of=/dev/sdb status=progress && sync
```

Comment:
@pyhalov 

Doing a `which dd` shows GNU version as default. So, I believe the coreutils version shipping with OI should support the status command. Perhaps I should say older versions of dd may not support that.  Admittedly I copied this line from the ARCH wiki;-) and tested it on Linux Mint 18. While it should work on the MATE release disk, I have not tested it on OI yet.